frontend/package.json

```diff
@@ -37,6 +38,7 @@
     "qs": "^6.9.7",
     "react": "17.0.2",
     "react-beautiful-dnd": "^11.0.0-beta.3",
+    "react-cookie": "^4.1.1",
```

Comment:
Can you coordinate with Álvaro? He's added a `setCookie` function in the `utilities/helpers` file.

Reply:
on it